core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/udf/python/DualInputPortsPythonUDFOpDescV2.scala

```diff
@@ -22,25 +22,28 @@ class DualInputPortsPythonUDFOpDescV2 extends OperatorDescriptor {
     defaultValue =
       "# Choose from the following templates:\n" +
         "# \n" +
-        "# from pytexera import *\n" +
+        "# import pytexera as pt\n" +
+        "# from overrides import overrides\n" +
+        "# from typing import Iterator, Optional\n" +
         "# \n" +
-        "# class ProcessTupleOperator(UDFOperatorV2):\n" +
+        "# \n" +
```

Comment:
Here I am adding a new line to fix the warning of single blank line between imports and class definitions.